src/main/java/core/basesyntax/Storage.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+
+    private Map<String, Integer> fruits = new HashMap<>();
```

Comment:
Follow the encapsulation principle. You have to hide your class elements with the help of access modifiers. You should make `fruits` private and provide public methods to interact with it.